### PR TITLE
Prototype frontend. Missing search button.

### PR DESCRIPTION
**Pull request by NodeReplacer**

### CODE REVIEW

**Comment by NodeReplacer** [comment]
Updated the pull request with functioning search button (that moves to a search page that doesn't have anything there except a hyperlink linking back to the home page, but still. We can fill in the search logic later.)

And also make it look better later.

**Review by BenignPhantasm — CHANGES_REQUESTED** [review]
Just a couple of changes

File: app/assets/stylesheets/home.css [hunk 1]
```diff
@@ -0,0 +1 @@
+@import 'materialize';
```
Comment (BenignPhantasm) [inline app/assets/stylesheets/home.css:1]: I'm not sure if this is right. Can you use one of the `material` buttons in the view just to prove that it's working properly?

**Review by BenignPhantasm — APPROVED** [review]
Nice work!

**Pull request merged**